README.md

```diff
@@ -12,46 +14,25 @@
 - JAVASCRIPT
 - VSCODE
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://aneallaryea100.github.io/Capstone/)
```

Comment:
- [ ] It's a good thing you updated the link to your live demo as requested by the previous code reviewer, but the link is broken, it is not loading the content of your page, please kindly fix that.

Reply:
Link to demo is now fixed